capstone/src/main/webapp/youtubeScript.js

```diff
@@ -174,3 +198,75 @@ function isLetter(character) {
     return false;
   }
 }
+
+async function getTrending(categoryId) {
+  trendingResponse = await fetch('/trending_servlet?videoCategoryId=' + categoryId,)
+  trendingResponseJson = await trendingResponse.json();
+  const trendingVideoIds = [];
+  for (const item in trendingResponseJson.items) {
+    const videoId = trendingResponseJson.items[item].id;
+    trendingVideoIds.push(videoId);
+  }
+  const commentsList = []
+  for (const id in trendingVideoIds) {
+    videoCommentList = await fetch('/youtube_servlet?videoID=' + trendingVideoIds[id],)
+    videoCommentListJson = await videoCommentList.json();
```

Comment:
Same here - we should declare these with `const`

Reply:
addressed in "spacing and address Daniel's nits"